src/loc/lcl/chs/diagnosticMessages/diagnosticMessages.generated.json.lcl

```diff
@@ -6588,11 +6588,11 @@
         </Str>
         <Disp Icon="Str" />
       </Item>
-      <Item ItemId=";Specify_library_files_to_be_included_in_the_compilation_Colon_6079" ItemType="0" PsrId="306" Leaf="true">
```

Comment:
please do not change the files under loc. they are managed by automated tools used by our translators. the new message will be shared with them automatically, and their tools will update it.

Reply:
Okay, @mhegazy.
Anyway, may I know more about this? Where is the tool which updates them?
Just curious, how are those files updated? Thank you.